src/python/CRABClient/ClientMapping.py

```diff
@@ -67,15 +67,17 @@
 }
 
 renamedParams = {
-    'General.transferOutput' : 'General.transferOutputs',
-    'General.saveLogs'       : 'General.transferLogs',
-    'Data.outlfn'            : 'Data.outLFN',
-    'Data.dbsUrl'            : 'Data.inputDBS',
-    'Data.publishDbsUrl'     : 'Data.publishDBS',
-    'Data.userInputFile'     : 'Data.userInputFiles',
-    'JobType.numcores'       : 'JobType.numCores',
-    'JobType.maxmemory'      : 'JobType.maxMemoryMB',
-    'JobType.maxjobruntime'  : 'JobType.maxJobRuntimeMin'
+    'General.transferOutput'          : 'General.transferOutputs',
+    'General.saveLogs'                : 'General.transferLogs',
+    'Data.outlfn'                     : 'Data.outLFN',
+    'Data.outLFN'                     : 'Data.outLFNDirBase'
+    'Data.dbsUrl'                     : 'Data.inputDBS',
+    'Data.publishDbsUrl'              : 'Data.publishDBS',
+    'Data.userInputFile'              : 'Data.userInputFiles',
+    'JobType.numcores'                : 'JobType.numCores',
+    'JobType.maxmemory'               : 'JobType.maxMemoryMB',
+    'JobType.maxjobruntime'           : 'JobType.maxJobRuntimeMin'
+    'JobType.allowNonProductionCMSSW' : 'JobType.allowUndistributedCMSSW'
```

Comment:
In fact, please put a comma at the end of this line too. That way if you add something to the end of the list, you don't have to add a comma generating a spurious diff. IIRC, this works in most languages but Javascript.